theme/theme.inc

```diff
@@ -84,6 +84,38 @@ function template_preprocess_islandora_basic_collection_wrapper(&$variables) {
 }
 
 /**
+ * Implements hook_theme().
+ */
+function theme_islandora_basic_collection_metadata_table_drag_components($vars) {
+  $element = $vars['element'];
+  drupal_add_tabledrag('islandora_basic_collection_meta_page_element_table', 'order', 'sibling', 'item-row-weight');
```

Comment:
Never sure here, but should possibly use [`drupal_html_id()`](https://api.drupal.org/api/drupal/includes!common.inc/function/drupal_html_id/7.x) to get a unique ID for a page, in the case there happen to be multiple rendered. Would have to make sure to use the same [ID below](https://github.com/Islandora/islandora_solution_pack_collection/pull/179/files#diff-8964427b21f6180b29573c0367f4b3e6R114), presumably.